manifests/charts/cloudcore/values.yaml

```diff
@@ -1,8 +1,3 @@
-# Default values for kubeedge.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-appVersion: "1.13.0"
-
 cloudCore:
   replicaCount: 1
   hostNetWork: true
```

Comment:
if this flag still keep, we still need to a choice?  I remember hostNotWork has a different working principle. Pls help to check it, thanks!

Reply:
I am not sure why hostnetwork is here, maybe it's for some special cases?
But I think it's okay to keep it here and make it possible for chart users to customize the values easily.
Of course, It would be better to add some comments here to avoid confusion, as I had before when seeing this.

Reply:
I think the older version is using the hostNetwork mode  on the bare meta vm.